packages/scripts/src/index.ts

```diff
@@ -1,13 +1,4 @@
-export {
```

Comment:
Please don't do this. Export from this file will become a gulp command, and I want it to be explicit.